deployment/v2/utils/updateVanillaGenesis.ts

```diff
@@ -290,6 +288,70 @@ async function updateVanillaGenesis(genesis, chainID, initializeParams) {
     expect(txObject.from).to.equal(ethers.recoverAddress(txObject.unsignedHash, txObject.signature as any));
     batch2.addRawTx(txInitializeBridge);
 
+    // deploy aggOracle if necessary
+    let globalExitRootUpdater;
+    let aggOracleImplementationAddress;
+    if (initializeParams.useAggOracleCommittee === true) {
+        const aggOracleCommitteeFactory = await ethers.getContractFactory('AggOracleCommittee');
+        // Get deploy transaction for agg oracle commitee
+        const deployAggOracle = await aggOracleCommitteeFactory.getDeployTransaction(gerProxy.address);
+
+        injectedTx.to = null;
+        injectedTx.data = deployAggOracle.data;
+        const txObjectDeployAggOracleImpl = ethers.Transaction.from(injectedTx);
+        expect(txObjectDeployAggOracleImpl.from).to.equal(
+            ethers.recoverAddress(
+                txObjectDeployAggOracleImpl.unsignedHash,
+                txObjectDeployAggOracleImpl.signature as any,
+            ),
+        );
+
+        const txDeployAggORacleCommitee = processorUtils.rawTxToCustomRawTx(txObjectDeployAggOracleImpl.serialized);
+        batch2.addRawTx(txDeployAggORacleCommitee);
+        aggOracleImplementationAddress = getContractAddress({
+            from: txObjectDeployAggOracleImpl.from,
+            nonce: injectedTx.nonce,
+        });
+
+        // Deploy proxy
+        /*
+         * deploy aggORacle proxy and initialize
+         */
+        const transparentProxyFactory = await ethers.getContractFactory(
+            '@openzeppelin/contracts4/proxy/transparent/TransparentUpgradeableProxy.sol:TransparentUpgradeableProxy',
+        );
+
+        // create initialize transaction
+        // Initialize aggOracle Manager
+        const initializeAggOracleCommittee = aggOracleCommitteeFactory.interface.encodeFunctionData('initialize', [
+            proxiedTokensManager, // TODO
```

Comment:
I think this is a good point to discuss, if using the `proxiedTokensManager` or a different address being set in the params.
If we set the `proxiedTokensManager` we will can not create sovereign chains with the token upgradeability "feature" disabled.

Reply:
i can add a new address to be the owner. AS i though we can always transfer it  aaand this should be a mutisig.
We can update it tho